requirements.txt

```diff
@@ -1,6 +1,6 @@
 aioamqp>=0.13.0
 aiohttp
-aioredis
+aioredis==2.0.0
```

Comment:
Please use `>=` here (we want to allow users of libmozevent to use never versions of aioredis, e.g. if 2.0.1 is released.